infrastructure/parameters/dev.parameters.json

```diff
@@ -28,6 +28,12 @@
     },
     "deploySubnets": {
       "value": true
+    },
+    "blobDeleteRetentionEnabled": {
+      "value": true
+    },
+    "blobDeleteRetentionDays": {
+      "value": 1
```

Comment:
Handy addition we probably dont want the 90 days on dev, may be worth setting to 7 days or so?

Reply:
Yeah was just setting to 1 until I've tested & then probs remove once enabled = false (see above).